fenix/app/src/main/java/org/mozilla/fenix/debugsettings/navigation/FenixDebugDrawerDestinations.kt

```diff
@@ -0,0 +1,35 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package org.mozilla.fenix.debugsettings.navigation
+
+import org.mozilla.fenix.R
+
+/**
+ * The navigation route for [DebugDrawerLandingScreen].
+ */
+const val DEBUG_DRAWER_ROUTE_LANDING_SCREEN = "landing_screen"
+
+/**
+ * The navigation route for [TabTools].
+ */
+const val DEBUG_DRAWER_ROUTE_TAB_TOOLS = "tab_tools"
+
+/**
+ * Creates a list of [DebugDrawerDestination]s for the debug drawer features in Fenix.
+ */
+fun generateDebugDrawerDestinations(
+    // Composable content dependencies will go here (e.g. browser store, the list of menu items, etc.)
```

Comment:
could add a todo here

Reply:
I initially opted to not do this, since I think we've been historically hesitant to committing todos. Regardless, this is temporary and was more of an explainer for peopl reviewing it.

Reply:
that's fair, the only reason I'd prefer a todo is that it makes it harder for the comment to get forgotten, but this is not a blocker